pkg/pgmodel/end_to_end_tests/drop_test.go

```diff
@@ -5,6 +5,7 @@ package end_to_end_tests
 
```

Comment:
Do we have an end-to-end test with an epoch abort? That should be doable end-to-end? Can we make sure that a retry works?

Reply:
e2e showing an abort is [here](https://github.com/timescale/promscale/pull/304/files#diff-59a0f996197a51fe1276a3e186e4ab912e5dc3673ee9bf2c0699549834cb1605R518-R521) getting a successful retry has to be done in the unit tests right now because we don't want to wait the cache refresh interval.